deploy/gk-deploy

```diff
@@ -476,10 +476,14 @@ fi
 
 if [[ ${LOAD} -eq 0 ]]; then
   if [[ "${CLI}" == *oc\ * ]]; then
-    eval_output "${CLI} process -p HEKETI_ADMIN_KEY=${ADMIN_KEY} -p HEKETI_USER_KEY=${USER_KEY} deploy-heketi | ${CLI} create -f - 2>&1"
+    oc_version = $(${CLI} version | grep "oc")
+    if [[ "${oc_version}" == *3.4.* ]]; then
```

Comment:
Thanks. Not sure depending on minor version will work for earlier versions of origin or openshift.  also, I was first thinking to have this option as a command line switch and a variable set, but then we are introducing a new option  which is not known to existing users.  so dropped it. but above looks better, i will update patch set.